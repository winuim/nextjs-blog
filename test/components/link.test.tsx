import React from 'react';
import {render} from '../testUtils';
import Link from '../../components/link';

const useRouter = jest.spyOn(require('next/router'), 'useRouter');

describe('Copyright component', () => {
  it('matches snapshot', () => {
    useRouter.mockImplementationOnce(() => ({
      pathname: '/home',
    }));
    const {asFragment} = render(<Link href="/home" />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
