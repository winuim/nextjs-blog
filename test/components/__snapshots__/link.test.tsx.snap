// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright component matches snapshot 1`] = `
<DocumentFragment>
  <a
    class="MuiTypography-root MuiLink-root MuiLink-underlineHover active MuiTypography-colorPrimary"
    href="/home"
  />
</DocumentFragment>
`;
